crates/cosmos/cosmos-integration-tests/tests/cosmos_integration_tests.rs

```diff
@@ -1,45 +1,53 @@
 #![recursion_limit = "256"]
 
+use std::env;
 use std::sync::Arc;
 
-use hermes_cosmos_chain_components::types::config::gas::dynamic_gas_config::DynamicGasConfig;
 use hermes_cosmos_integration_tests::contexts::binary_channel::setup::CosmosBinaryChannelSetup;
-use hermes_cosmos_integration_tests::contexts::bootstrap::CosmosBootstrap;
-use hermes_cosmos_integration_tests::init::init_test_runtime;
-use hermes_cosmos_relayer::contexts::build::CosmosBuilder;
+use hermes_cosmos_integration_tests::init::{build_gaia_bootstrap, init_test_runtime, TestPreset};
 use hermes_error::types::Error;
 use hermes_ibc_test_suite::tests::transfer::TestIbcTransfer;
 use hermes_test_components::setup::traits::run_test::CanRunTest;
+
 use ibc_relayer_types::core::ics24_host::identifier::PortId;
 
 #[test]
 fn cosmos_integration_tests() -> Result<(), Error> {
     let runtime = init_test_runtime();
 
-    let builder = Arc::new(CosmosBuilder::new_with_default(runtime.clone()));
-
-    // TODO: load parameters from environment variables
-    let bootstrap = Arc::new(CosmosBootstrap {
-        runtime: runtime.clone(),
-        cosmos_builder: builder,
-        should_randomize_identifiers: true,
-        chain_store_dir: "./test-data".into(),
-        chain_command_path: "gaiad".into(),
-        account_prefix: "cosmos".into(),
-        staking_denom_prefix: "stake".into(),
-        transfer_denom_prefix: "coin".into(),
-        genesis_config_modifier: Box::new(|_| Ok(())),
-        comet_config_modifier: Box::new(|_| Ok(())),
-        dynamic_gas: Some(DynamicGasConfig::default()),
-    });
-
-    let setup = CosmosBinaryChannelSetup {
-        bootstrap_a: bootstrap.clone(),
-        bootstrap_b: bootstrap,
-        create_client_settings: Default::default(),
-        init_connection_options: Default::default(),
-        init_channel_options: Default::default(),
-        port_id: PortId::transfer(),
+    let test_preset = env::var("TEST_PRESET")
+        .unwrap_or_else(|_| "CosmosToCosmos".to_string())
+        .parse::<TestPreset>()?;
+
+    let setup = match test_preset {
```

Comment:
This should be done inside a function that returns two `CosmosBootstrap`s.

Reply:
Something like this 2217bdd69e9882a7d19f52dcba1c879d3847f1e3 ?